src/views/Login/Shards.js

```diff
@@ -86,129 +36,58 @@ class Shards extends React.Component {
-            <ValidatedSigil
-              className={'tr-0 mt-05 mr-0 abs'}
-              patp={pointName}
-              size={68}
-              margin={8}
-            />
-          </PointInput>
-
-          <ShardInput
-            className="mono mt-8"
-            prop-size="md"
-            prop-format="innerLabel"
-            type="text"
-            name="shard1"
-            placeholder={`e.g. ${phTick}`}
-            value={shard1}
-            onChange={inp => this.handleShardInput(SHARDS.SHARD1, inp)}>
-            <InnerLabel>{'Shard 1'}</InnerLabel>
-          </ShardInput>
-
-          <ShardInput
-            className="mono mt-8"
-            prop-size="md"
-            prop-format="innerLabel"
-            type="text"
-            name="shard2"
-            placeholder={`e.g. ${phTick}`}
-            value={shard2}
-            onChange={inp => this.handleShardInput(SHARDS.SHARD2, inp)}>
-            <InnerLabel>{'Shard 2'}</InnerLabel>
-          </ShardInput>
-
-          <ShardInput
-            className="mono mt-8"
-            prop-size="md"
-            prop-format="innerLabel"
-            type="text"
-            name="shard3"
-            placeholder={`e.g. ${phTick}`}
-            value={shard3}
-            onChange={inp => this.handleShardInput(SHARDS.SHARD3, inp)}>
-            <InnerLabel>{'Shard 3'}</InnerLabel>
-          </ShardInput>
-
-          <InputCaption>
-            {'If your wallet requires a passphrase, you may enter it below.'}
-          </InputCaption>
-
-          <Input
-            className="pt-8"
-            prop-size="md"
-            prop-format="innerLabel"
-            name="passphrase"
-            type="password"
-            value={passphrase}
-            autocomplete="off"
-            onChange={this.handlePassphraseInput}>
-            <InnerLabel>{'Passphrase'}</InnerLabel>
-          </Input>
-
-          <Button
-            disabled={!ready}
-            className={'mt-8'}
-            prop-size={'lg wide'}
-            // prop-color={this.buttonTriState(wallet)}
-            onClick={() =>
-              this.walletFromShards(
-                shard1,
-                shard2,
-                shard3,
-                pointName,
-                passphrase
-              )
-            }>
-            {'Unlock Wallet →'}
-          </Button>
-
-          <Button
-            className={'mt-4'}
-            prop-size={'xl wide'}
-            disabled={Nothing.hasInstance(wallet)}
-            onClick={this.props.loginCompleted}>
-            {'Continue →'}
-          </Button>
-        </Col>
-      </Row>
-    );
-  }
+  };
+
+  const shards = [shard1, shard2, shard3];
+  const ready = shards.filter(x => x !== '').length > 1;
+
+  return (
+    <View>
+      Enter your point and at least two of your three Urbit master ticket shards
+      here. The index of the input field needs to match the index of the shard.
+      <PointInput
```

Comment:
check out the PointInput + InputSigil from ViewPoint for how to integrate a validated sigil into here

Reply:
Shouldn't that just be part of PointInput though?

Reply:
@Fang- yeah, good idea!

Reply:
Hmm, I tried taking a stab at this, but couldn't get it to work. No console error, just didn't render... Also probably dealt with the error, pass and focussed props in `ViewPoint` in the wrong/inelegant way. Could we `useState` inside of `components/Inputs`, to factor that out?

Reply:
hmm, I'll take a look at it once we get my `useForm` changes into mino

Reply:
nvm, I was overlooking something dumb!

This isn't as pretty everywhere as it is in `ViewPoint` because not everyone uses the `error` etc props yet. I'll try poking at it a bit, but take a look at the commit I just pushed and lmk your thoughts.